Change Link to Button

kellyramirezz opened the pull request:
I changed the link to a button w/ a white text and same bg color as the border. 
![Screenshot 2025-06-06 122502](https://github.com/user-attachments/assets/04faa7d3-acd8-4b6e-8341-1e10676e212c)

kavinphab commented on src/components/about/homePage.tsx:
```diff
@@ -80,8 +80,13 @@ const HomePage = () => {
               <p>Find us on the Highlander Link:</p>
-              <a href="https://highlanderlink.ucr.edu/organization/rbrains">
-                https://highlanderlink.ucr.edu/organization/rbrains
+              <a
```
Use next/link: put this at the top import Link from "next/link"; to import link then change the <a> to <Link> as this will allow for client side navigation

VacaTheCow commented on src/components/about/homePage.tsx:
```diff
@@ -80,9 +81,12 @@ const HomePage = () => {
-              <a href="https://highlanderlink.ucr.edu/organization/rbrains">
-                https://highlanderlink.ucr.edu/organization/rbrains
-              </a>
+              <Link
```
Instead of link use a button from our ReusableButton.tsx. It's already imported:

<img width="371" alt="image" src="https://github.com/user-attachments/assets/dde0f5d8-3ac6-427b-b768-68fbbf80c1f2" />


and use it like this:

<img width="261" alt="image" src="https://github.com/user-attachments/assets/8aaced9b-23b4-481f-80a6-db97de8d0030" />

It takes in three parameters: Padding, textsize, and the border color. This way all our buttons are consistent throughout the site.

VacaTheCow reviewed (changes requested):
I noticed Kavin asked for your to use next link but we have our ReusableBox.tsx to cover those button, just put in the parameter (see my comment for exact changes). Sorry for the confusion :)

VacaTheCow commented:
Last thing, the button should link to here:

https://highlanderlink.ucr.edu/organization/rbrains

kellyramirezz commented:
I fixed the merge conflicts and changed to the reusable button. 
![Screenshot 2025-06-10 210225](https://github.com/user-attachments/assets/6f166e9c-ce66-45a0-be08-6c2fa6b32112)

VacaTheCow reviewed (approved):
WHOOOOO good job

VacaTheCow reviewed (approved):
approved
